api/db-cdr/generate-cdr/run-achilles-queries.sh

```diff
@@ -790,4 +790,30 @@ group by dbd.concept_id,o.observation_source_concept_id,o.value_as_number,stratu
 #Change concept ids from 0 to null
 bq --quiet --project=$BQ_PROJECT query --nouse_legacy_sql \
 "update \`${WORKBENCH_PROJECT}.${WORKBENCH_DATASET}.db_domain\`
-set concept_id=null where concept_id=0"
\ No newline at end of file
+set concept_id=null where concept_id=0"
+
+# Domain participant counts
+echo "Getting domain participant counts"
+bq --quiet --project=$BQ_PROJECT query --nouse_legacy_sql \
+"insert into \`${WORKBENCH_PROJECT}.${WORKBENCH_DATASET}.achilles_results\`
```

Comment:
We can join the db_domain table but since the fetch is from different tables (observation, measurement, drug_exposure and procedure_occurrence) i thought individual queries does the good job. Either way even if we join to the db_domain table and fetch the domain id we have to add whatever table suffix to fetch from.

Reply:
Good point. This looks good then. Keep it simple.